src/components/wms/InformationPanel.vue

```diff
@@ -0,0 +1,204 @@
+          :prepend-icon="timeIcon"
+        >
+        </v-list-item>
+        <v-list-group>
+          <template v-slot:activator="{ props }">
+            <v-list-item
+              v-bind="props"
+              title="Color scales"
+              :prepend-icon="colorScalesIcon"
+            ></v-list-item>
+          </template>
+          <v-list-item
+            v-for="(style, index) in props.styles"
+            :key="index"
+            :title="style.title"
+            :subtitle="style.name"
+            @click="onStyleClick(style)"
+            :class="{ selected: isSelected(style) }"
+          >
+          </v-list-item>
+          <v-list-item v-if="mutableColorScaleRange" :prepend-icon="rangeIcon">
+            <v-row align="center">
+              <v-col cols="6">
+                <v-text-field
+                  v-model.number="mutableColorScaleRange.min"
+                  label="Min"
+                  outlined
+                  clearable
+                  :rules="[
+                    rules.required,
+                    rules.biggerThanZero,
+                    rules.smallerThanMax,
+                  ]"
+                ></v-text-field>
+              </v-col>
+              <v-col cols="6">
+                <v-text-field
+                  v-model.number="mutableColorScaleRange.max"
+                  label="Max"
+                  outlined
+                  clearable
+                  :rules="[rules.required, rules.biggerThanMin]"
+                ></v-text-field>
+              </v-col>
+            </v-row>
+            <v-btn
+              color="primary"
+              variant="text"
+              @click="changecolorScaleRange"
+            >
+              Submit
+            </v-btn>
+          </v-list-item>
+        </v-list-group>
+        <v-list-item v-if="props.completelyMissing">
+          Wms layer is completely missing
+        </v-list-item>
+      </v-list>
+    </v-menu>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import { DateTime } from 'luxon'
+import { Style } from '@deltares/fews-wms-requests'
+import { ref } from 'vue'
+
+interface Props {
+  layerTitle: string
+  currentTime: Date | null
+  forecastTime: Date | null
+  styles: Style[] | null
+  completelyMissing: boolean | null
+  firstValueTime: Date | null
+  lastValueTime: Date | null
+  colorScaleRange: { min: number; max: number } | null | undefined
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  layerTitle: '',
+  currentTime: null,
+  forecastTime: null,
+  styles: null,
+  completelyMissing: null,
+  firstValueTime: null,
+  lastValueTime: null,
+  colorScaleRange: null,
+})
+
+const emit = defineEmits(['style-click', 'color-scale-range-change'])
+const layersIcon = 'mdi-layers'
+const timeIcon = 'mdi-clock-time-four-outline'
+const rangeIcon = 'mdi-layers-edit'
+const colorScalesIcon = 'mdi-palette'
+const selectedStyle = ref<Style | null>(null)
```

Comment:
Same here for example where we can do
```suggestion
const selectedStyle = ref<Style>()
```
And then check for undefined in isSelected.